### ISSUE TITLE
setup clarification

### ISSUE THREAD

**Issue by odysseyalive** [denis303/codeigniter4-recaptcha#4]
This might be obvious to most, but I'm still making friends with CI4 and PSR in general.

The only thing in the documentation that I was missing to make this work for me was adding the namespace in Autoload:
```
'Denis303\ReCaptcha' => APPPATH . 'ThirdParty/vendor/denis303/codeigniter4-recaptcha'
```

Great job on this BTW!  This saves a lot of work.

**Comment by dadreggors** [comment]
Yes, pleas clarify where in Autoload file to add this line and what is the proper line to add. I have seen several versions of this line using ROOTPATH or APPPATH and different text on left or right or APPPATH/ROOTPATH as well.

**Comment by odysseyalive** [comment]
> Yes, pleas clarify where in Autoload file to add this line and what is the proper line to add. I have seen several versions of this line using ROOTPATH or APPPATH and different text on left or right or APPPATH/ROOTPATH as well.

It's been a while since I've used this. I believe I was referring to adding it as an entry to $autoload['libraries'].
